docs/install/iso-install.md

```diff
@@ -27,10 +27,10 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 1. Choose the installation device on which the Harvester cluster will be installed
       - Note: By default, Harvester uses [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) partitioning schema for both UEFI and BIOS. If you use the BIOS boot, then you will have the option to select [MBR](https://en.wikipedia.org/wiki/Master_boot_record).
    ![iso-install-disk.png](./assets/iso-install-disk.png)
-1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or a static assigned a static one.
+1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or a static assigned a static one <small>(Note: The NODE IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, user must make sure the DHCP server always offers the same IP for the same NODE. A changed NODE IP will cause the related NODE can not join the cluster, or even break the clusterP)</small>.
```

Comment:
`the IP address can either be configured via DHCP or a static assigned a static one` 
-> 
`the IP address can be configured via DHCP or a static assigned one`